products.meta

```diff
@@ -197,6 +197,12 @@
       "type": "product",
       "dependencies": ["automate"],
       "services": []
+    },
+    {
+      "name": "sample-data",
+      "type": "product",
+      "dependencies": ["automate"],
+      "services": []
```

Comment:
I think we'll want to list the new services in this service array as `chef/sample-data-service`